### DISCUSSION TITLE
q-input long hint cover another field

### DISCUSSION THREAD

**Discussion by htsoft** [quasarframework/quasar#11404] [General - Components / Directives / etc] [answered]
Hi all,

In a q-input I'm using the hint property to show a long hint (it's required). But when displayed on a mobile device, the hint will cover the lower control.

![image](https://user-images.githubusercontent.com/832779/142591060-4b71ac60-7f6e-49d0-8edc-41088785a585.png)

Here is the code:
```
<q-input
   v-model="data.university.degreeQualif"
   label="University degree qualification (BA, BS, BC, BFA, BE in ...)"
   class="col-xs-12 col-md-12"
   outlined
   dense
   style="margin-top:10px; margin-bottom:10px"
   hint="Please specify name of the course or the degree you obtained in case you optioned 'Other' if the previous list"
 />
 <q-input filled v-model="unStart" mask="##/##/####" label="University entered date" dense style="margin-top:10px">
    <template v-slot:append>
       <q-icon name="event" class="cursor-pointer">
          <q-popup-proxy ref="qDateProxy4" cover transition-show="scale" transition-hide="scale">
              <q-date v-model="unStart" mask="DD/MM/YYYY">
                 <div class="row items-center justify-end">
                   <q-btn v-close-popup label="Close" color="primary" flat />
                  </div>
               </q-date>
            </q-popup-proxy>
          </q-icon>
        </template>
  </q-input>

```

How can i solve this issue?

Kindly
Roberto

**Comment by MilosPaunovic** [comment, marked as answer]
You can make use of `hint` slot of `QInput`

[https://codepen.io/milospaunovic/pen/QWMojpm?editors=101](https://codepen.io/milospaunovic/pen/QWMojpm?editors=101)

**Reply by htsoft** [reply]
I've solved it using @media in the style section. But I will give a test to your suggestion.

Thank you.

**Reply by mickey-bimlive** [reply]
@MilosPaunovic - can you post an example how you use @media for this problem with hints? I have the same problem with many of my hints, and would like to avoid to convert all affected q-inputs to use v-slots.

**Reply by MilosPaunovic** [reply]
@htsoft is the one to provide you with his media solution, if he's kind enough 🙂

**Reply by htsoft** [reply]
@mickey58github in the component style section I've hadded this class:
```
.below-related-margin
  @media (max-width:420px)
    margin-bottom:40px
  @media (min-width:421px)
    margin-bottom:20px

```

and added the class to the component.

**Reply by mickey-bimlive** [reply]
Thanks - I'll try this out.
